pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -357,37 +357,34 @@ def visit_const(self, node: nodes.Const) -> None:
             ):
                 self._detect_replacable_format_call(node)
 
+    # pylint: disable = too-many-branches
```

Comment:
We need to refactor first I think. This function became too big to understand easily. (It's also better to do the ``parent = node.parent`` refactor as a standalone so we can revert and understand the change easily).